src/vibrato.ts

```diff
@@ -68,8 +69,12 @@ export class Vibrato extends Modifier {
   setVibratoWidth(width: number): this {
     this.renderOptions.width = width;
     this.text = String.fromCodePoint(this.renderOptions.code);
+    const myWidth = this.getWidth();
+    if (!myWidth) {
```

Comment:
I'm gonna go ahead and merge this PR, but in the future I think we should test for zero with === 0 instead of !.

if (!boo) should probably be reserved for boolean variables, and at least I tend to read them as booleans at first glance.

Reply:
It also checks for undefined/null.
But yes, if undefined isn't possible it basically checks === 0.
If the idea is that the width should be positive, I would check >= 0, because we probably also don't want negative width.